packages/kotti-ui/source/kotti-inline-edit-2/KtInlineEdit2.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+	<div :class="objectClass">
+		<label class="form-label" v-text="label" />
+		<div v-if="isEditing">
+			<component
+				:is="inputComponent"
+				v-bind="$attrs"
+				class="form-input"
+				type="text"
+				:value="currentValue"
+				@change="handleChange"
```

Comment:
i think this also triggers if you press `Enter` (return key) while you're in the `textarea`

Reply:
in the `text`*